### DISCUSSION TITLE
Some features don't work (kohya_ss)

### DISCUSSION THREAD

**Discussion by camilleif** [bmaltais/kohya_ss#1416] [Q&A]
Hello !

I'm on macOS, I managed to install kohya_ss but some features don't work. Like the open, save, load button for example.

Here's the message I get:

Warning: LD_LIBRARY_PATH environment variable is not set.
Certain functionalities may not work correctly.
Please ensure that the required libraries are properly configured.
 
If you use WSL2 you may want to: export LD_LIBRARY_PATH=/usr/lib/wsl/lib/
 

11:51:02-068697 INFO     Version: v21.8.7                                       
                                                                                
11:51:02-079872 INFO     Using CPU-only Torch                                   
11:51:04-592717 INFO     Torch 2.0.0                                            
11:51:04-594048 WARNING  Torch reports CUDA not available                       
11:51:04-595099 INFO     Verifying modules instalation status from              
                         /Users/kamilifrah/kohya_ss/requirements_macos_amd64.txt
                         ...                                                    
11:51:04-598165 INFO     Installing package: torch==2.0.0 torchvision==0.15.1 -f
                         https://download.pytorch.org/whl/cpu/torch_stable.html 
11:51:09-029056 INFO     Installing package: xformers bitsandbytes==0.41.1      
11:51:21-217225 ERROR    Error running pip: install --upgrade xformers          
                         bitsandbytes==0.41.1                                   
11:51:21-224685 INFO     Verifying modules instalation status from              
                         requirements.txt...                                    
11:51:23-935682 INFO     headless: False                                        
11:51:23-940838 INFO     Load CSS...                                            
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.

Do you know how to solve the problem?

Thank you !

**Comment by tomaebi** [comment]
Hey ! 
Did you finally manage to work this out? I have the same issue so I'd be curious...
